py/mattak/backends/pyroot/dataset.py

```diff
@@ -22,7 +22,13 @@
 cppyy.cppdef(" bool is_nully(void *p) { return !p; }")
 
 def isNully(p):
-    return p is None or ROOT.AddressOf(p) == 0 or cppyy.gbl.is_nully(p)
+    if isinstance(p, str):
+        return p is None
```

Comment:
this always returns true

Reply:
but good catch, cosmin and I fixed in our branches already but not yet in the main